src/tests/ftest/util/dmg_utils.py

```diff
@@ -546,7 +546,7 @@ def storage_scan(self):
         self.sub_command_class.set_sub_command("scan")
         return self._get_result()
 
-    def storage_format(self):
+    def storage_format(self, reformat=False):
```

Comment:
It's better to add the description of reformat parameter and maybe explain the effect of turning it on.

Reply:
Ok, will do.